index.js

```diff
@@ -34,11 +35,11 @@ module.exports = {
     // Allow to use `_` in functions and variable names
     'no-underscore-dangle': OFF,
 
-    // Set the max length of a line to 200 characters
+    // Set the max length of a line to 120 characters
```

Comment:
I feel 200 is too long - I've been accustomed to conventions of 100-120 limits. Airbnb itself has a 100 limit, so I think 120 will be fair for us.